sympy/matrices/common.py

```diff
@@ -2190,6 +2190,26 @@ def _eval_pow_by_recursion(self, num):
 
         return a.multiply(b)
 
+
+
```

Comment:
Why all the blank lines? Other functions here are only separated by a single blank line